helm/app-updater-app/values.yaml

```diff
@@ -0,0 +1,4 @@
+
+app_repo: docs
+app_name: docs-app"
+schedule: "*/30 * * * *"
```

Comment:
Missing parameter `clusterID` used in `configmap.yaml` line 11 and onward

Reply:
But cluster ID comes from cluster values configmap, right?
```
» k get cm c68pn-cluster-values -oyaml                        (giantswarm-gollum/c68pn)
apiVersion: v1
data:
  values: |

    clusterID: c68pn
```

Reply:
In the mentioned line there is `{{ .Values.clusterID }}` which in my understanding templates the value from key `clusterID` in the `values.yaml` right?

Reply:
done